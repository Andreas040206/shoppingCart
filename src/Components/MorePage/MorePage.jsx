import MorePageStyles from "./MorePage.module.css";
import RocketImgSrc from "../../assets/rocket.gif";

export const MorePage = () => {
  return (
    <div className={MorePageStyles.con}>
      <img className={MorePageStyles.img} src={RocketImgSrc} />
      <span className={MorePageStyles.text}>Sorry, nothing here...</span>
    </div>
  );
};
