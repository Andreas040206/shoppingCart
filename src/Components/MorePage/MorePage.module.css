.con {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: calc(100vh - 60px);
  gap: 30px;
}

.img {
  height: 150px;
}

.text {
  font-size: 24px;
}
