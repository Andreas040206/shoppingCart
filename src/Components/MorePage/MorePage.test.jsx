import { describe, it, expect } from "vitest";
import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";

import { MorePage } from "./MorePage";

describe("Testing the MorePage", () => {
  it("Loads things correctly", () => {
    const { container } = render(<MorePage />);
    expect(container).toMatchSnapshot();
  });
});
