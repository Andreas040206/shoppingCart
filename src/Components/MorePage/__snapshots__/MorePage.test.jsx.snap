// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Testing the MorePage > Loads things correctly 1`] = `
<div>
  <div
    class="_con_3361f4"
  >
    <img
      class="_img_3361f4"
      src="/src/assets/rocket.gif"
    />
    <span
      class="_text_3361f4"
    >
      Sorry, nothing here...
    </span>
  </div>
</div>
`;
